lib/Conversion/TritonGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -28,6 +28,8 @@ static CUtensorMapDataType getCUtensorMapDataType(Type ty) {
     return CUtensorMapDataType::CU_TENSOR_MAP_DATA_TYPE_BFLOAT16;
   } else if (ty.isF32()) {
     return CUtensorMapDataType::CU_TENSOR_MAP_DATA_TYPE_FLOAT32;
+  } else if (ty.isFloat8E5M2()) {
+    return CUtensorMapDataType::CU_TENSOR_MAP_DATA_TYPE_UINT8;
```

Comment:
what about the other FP8 data-types?

Reply:
Hi! I have just added other FP8 types, and people can use torch.float8_e4m3fn now :)

Reply:
I think we also want to support this for INT8. Can't we just check that the width of the data-type is 1B and use UINT8 if that's the case?

Reply:
Yes, I think we can do that!